app/jobs/generate_pdf_job.rb

```diff
@@ -80,4 +83,15 @@ def pdf_path(data_dir:, file_name:)
   def pdf_file_name(file_name:)
     file_name.sub(/(\.xml|\.pdf)?$/i, '').concat('.pdf').to_s
   end
+
+  def valid_pdf?(pdf_file_path:)
+    pdf = PDF::Reader.new(pdf_file_path)
```

Comment:
I think we'll get an `ArgumentError` error here if we try to read a PDF that isn't yet generated or wasn't generated? Maybe add a guard clause at the top that returns false if the file doesn't exist?